packages/xo-vmdk-to-vhd/README.md

```diff
@@ -16,9 +16,9 @@ To convert a VMDK stream to a Fixed VHD stream without buffering the entire inpu
 
 ```js
 import convertFromVMDK from 'xo-vmdk-to-vhd'
-import { createReadStream, createWriteStream } from 'fs'
+import {createReadStream, createWriteStream} from 'fs'
```

Comment:
Why this change?

Please reformat with latest version of Prettier, installed in this project.